test_conformance/api/test_kernel_private_memory_size.cpp

```diff
@@ -0,0 +1,58 @@
+#include "harness/errorHelpers.h"
+#include "harness/typeWrappers.h"
+#include <iostream>
+#include <vector>
+#include "procs.h"
```

Comment:
is this header needed?

Reply:
These headers are all needed.

Reply:
<vector> needs changed to <array> now I guess